text/0012-cva.md

```diff
@@ -0,0 +1,161 @@
+
+A rational risk mitigation strategy dictates that validators collaborate.  Timely information sharing over well defined communication channels is the backbone of such collaboration. 
+Validators operating in isolation are disempowered as compared to collaborative ad-hoc groups of validators.  However ad-hoc groups tend to coalesce into political cliques focussing upon narrow tactical objectives at the expense of coherent strategic objectives.  This strategic incoherence is to the detriment of the network's long-term viability.
+
+An ad-hoc landscape is unfit for the purpose of nurturing and stewarding a successful crypto-economic network.  It is thus recommended that an association, hereby called the Casper Validator Association, is formally established.  Such an association will be endowed with financial capital and reputational credibility.  It seems logical to incorporate the association in either Singapore or Zug, Switzerland.  
+
+Association membership incurs both duties & responsibilities and all genesis validators will be required to be members of this association.  To focus the minds of association members the initial objectives will be to:
+
+- draw up legally binding articles of association;
+- draw up initial constitution;
+- establish secure, auditable communication channels (see below);
+- establish a VDAO (see below);
+- establish a certification program (see below);
+
+### 2. Establish secure, auditable communication channels
+
+Validators need secure communication channels over which sensitive information can be transmitted and/or shared.  Such information may be sensitive in an operational sense (e.g. detailing a bug), or in a business sense (e.g. token movement disclosures).  Such information may originate from multiple sources simultaneously (e.g. errors showing up in logs) or from a single source in isolation (e.g. intent to leave the association).  
+
+The choice of communication provider(s) is an important one.  Any choice needs to be:
+
+- appropriate to the channel context;
+- trusted by the validator community;
+- available across differing OS platforms;
+- available across nation-state firewalls;
+- auditable in the event of disputes;
+
+Social media platforms are public-domain and hence by definition auditable, however they are inappropriate for communication of security and/or price sensitive information.  For such 'internal' communications it is advised to setup a dedicated communications platform using open-source products such as matrix.
+
+Auditabilty is a controversial subject that requires a form of mediation protocol.  I.E. a well-defined legally binding process must exist so that a mandated auditor can gain access to the subset of historical communication data pertinent to the  auditing context.  Initially this feature will probably not be supported as it will require consensus amongst validators following a probably protracted debate. 
+
+### 3. Establish a VDAO - i.e. validator DAO 
```

Comment:
Keep it voluntary, light and self-defined, beyond validator education and onboarding.